SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json

```diff
@@ -815,7 +815,7 @@
     },
     {
       "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_Quality_Sensing_Device;1",
-      "OutputDtmi": "dtmi:com:willowinc:AirQualitySensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:Device;1",
```

Comment:
Hey Jason, when updating a mapping, please also update the version numbers in the csproj so we can pick up the changes in the next release.

Reply:
Hi @joebeernink, I just updated the related csproj: https://github.com/Azure/opendigitaltwins-tools/pull/98/commits/727b1312780a2fd56844710db0f3c69fe719dc26
Let me know if that's correct. Otherwise, a specific pointer would be appreciated. Thanks!